Resources/Prototypes/Entities/Objects/Devices/payload.yml

```diff
@@ -76,3 +76,19 @@
         enum.ChemicalPayloadFilledSlots.Left: payload-chemical-left
         enum.ChemicalPayloadFilledSlots.Right: payload-chemical-right
         enum.ChemicalPayloadFilledSlots.Both: payload-chemical-armed
+
+- type: entity
+  name: flash payload
+  parent: BasePayload
+  id: FlashPayload
+  description: a flash payload , it only have one use
```

Comment:
Grammar wise this should be "has" instead of "have", but maybe just reword it to something like "A single-use flash payload.", which is a bit more concise.

Reply:
I will change that